docs/release-notes/next/PT-165980068-use-trace_runner.md

```diff
@@ -0,0 +1 @@
+* Add support for using the [trace_runner](https://github.com/uwiger/trace_runner) wrapper in test suites for better debugging.
```

Comment:
Is this change meant to be user oriented? 🤔 If so there should be some doc.

I looks to me this is developer-only, not visible to users, hence shall not be present in release notes.

Reply:
@lucafavatella Release notes mention removed.